.ddev/commands/web/install

```diff
@@ -8,7 +8,11 @@ drush site:install standard --account-name=admin --account-pass=admin --site-nam
 drush en admin_toolbar admin_toolbar_tools config_inspector devel
 
 # Install eslint dependencies
-yarn --cwd $DDEV_DOCROOT/core install
+# The "--cwd" argument doesn't work with Yarn 4.4, because the global version
+# is returned as 1.22.22 when Yarn starts, before switching directories.
+cd $DDEV_DOCROOT/core
+yarn install
+cd -
```

Comment:
@byrond This change suggests a fail elsewhere. How do I trigger that failure?

Reply:
This happened during `ddev install`:
```
yarn install v1.22.22
[1/5] Validating package.json...
error Drupal@: The engine "yarn" is incompatible with this module. Expected version ">= 4.1.1". Got "1.22.22"
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

I believe it's because the required Yarn version isn't known until changing into the core directory, after Yarn has already started with the global version installed in the container.